MREUnityRuntime/MREUnityRuntimeLib/App/IMixedRealityExtensionApp.cs

```diff
@@ -133,5 +139,16 @@ public interface IMixedRealityExtensionApp
         /// </summary>
         /// <param name="actorId"></param>
         void OnActorDestroyed(Guid actorId);
+
+        /// <summary>
+        /// Send the event, using the current protocol
+        /// </summary>
+        /// <param name="update"></param>
+        void SendEvent(Payload update);
```

Comment:
Sending events and working with payloads should not be exposed to host app implementations